tests/phpunit/tests/blocks/editor.php

```diff
@@ -31,9 +31,19 @@ public function set_up() {
 
 		global $post_ID;
 		$post_ID = 1;
+
+		global $wp_scripts, $wp_styles;
+		$this->original_wp_scripts = $wp_scripts;
+		$this->original_wp_styles  = $wp_styles;
+		wp_scripts();
+		wp_styles();
```

Comment:
This will ensure the globals are reinstantiated 
```suggestion
		$wp_scripts = null;
		$wp_styles = null;
		wp_scripts();
		wp_styles();
```